test/unit/range/iterator_test_template.hpp

```diff
@@ -26,6 +26,7 @@ struct iterator_fixture : public ::testing::Test
                                                                // std::bidirectional_iterator_tag
                                                                // std::random_access_iterator_tag
 
+    // TODO: sized_sentinel_for is another property that we could support
```

Comment:
Did you open a card/issue? I would avoid TODO comments in the code.

Reply:
No, not sure where to add it.

Reply:
Smarty pants :D But I'm not sure if I am the right person to ask. I'm still not sure if I understood all of the rules in the new project planning. If you'd ask my personal opinion, I still like the "one project per module structure" and I would keep those kind of todos in the respective module as a card or issue